app/models/concerns/orchestration/tftp.rb

```diff
@@ -209,4 +208,10 @@ def each_unique_feasible_tftp_proxy
     end
     results.all?
   end
+
+  def local_boot_template_name(kind)
+    key = "local_boot_#{kind}"
+    host_param = HostParameter.find_by(:reference_id => host.id, :name => key)
```

Comment:
Best to look in `host.host_params`, in order to get all possible values (from hostgroup, global parameters, inherited parameters, etc...)